modules/arm_plugin/armplg_build.sh

```diff
@@ -132,11 +131,15 @@ cmake -DOpenCV_DIR=$STAGING_DIR/opencv/cmake -DENABLE_OPENCV=OFF \
       -DCMAKE_EXE_LINKER_FLAGS=-Wl,-rpath-link,$STAGING_DIR/opencv/lib -DCMAKE_INSTALL_LIBDIR=lib \
       -DENABLE_SSE42=OFF -DENABLE_MYRIAD=ON -DENABLE_GNA=OFF -DCMAKE_BUILD_TYPE=$BUILD_TYPE \
```

Comment:
* `-DENABLE_SSE42=OFF` is OFF by default for ARM. Can be removed 
* `-DENABLE_PROFILING_ITT=OFF` the same
* `-DENABLE_BEH_TESTS=ON` relates to deprecated tests, while ARM plugin does not have them
* `-DENABLE_CLDNN_TESTS=OFF` is OFF since GPU is off for ARM. Can be removed
* `-DENABLE_GNA=OFF` is not enabled for ARM at all. Can be removed
* `-DCMAKE_CXX_FLAGS=-latomic` why do we need it? all flags must a part of cmake and extra flags must not be explicitly passed
* `-DCMAKE_INSTALL_LIBDIR=lib` why do we need it?
* `-DCMAKE_INSTALL_PREFIX=$STAGING_DIR` is specified 2 times: cmake configure, cmake install. Let's keep only once

Reply:
@terfendail could you refresh my memory, why did we add `CMAKE_CXX_FLAGS=-latomic` to the script?
I believe we may remove all other flags mentioned by Ilya (and, probably, this one as well).

Reply:
@ilya-lavrenov I removed all flags you mentioned

Reply:
`latomic` was added due to [this](https://github.com/openvinotoolkit/openvino/issues/2956)
Thus I can't say for sure whether it's still required or remember the exact use-case that cause the issue.

Reply:
I believe the proper fix should be in cmake like this https://github.com/opencv/opencv/pull/15353/files

Reply:
All flags mentioned by Ilya were removed.